packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -246,12 +247,9 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
         }
     }
 
-    private rejectDeferredRealize(reason: string): never {
-        const error = new Error(reason);
-        // Error messages contain package names that is considered Personal Identifiable Information
-        // Mark it as such, so that if it ever reaches telemetry pipeline, it has a chance to remove it.
-        (error as any).containsPII = true;
-        throw error;
+    private rejectDeferredRealize(reason: string, packageName?: string): never {
+        const pii = { packageName };
```

Comment:
This code right here assumes it's PII, but it's not, unless we have 3rd party code in container.
I think categorization should tell it is (i.e. we should have some number of categories like package name), not how it will be treated in specific environment.

Reply:
Yep for sure. I was trying to do a small incremental change (it's already wrongly called out as PII) but I see your comment below and will try to accommodate this now.